useful-resources/press-kit.html.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Press kit
+---
+You can find useful brand resources in the Nebulab's [Press kit](https://drive.google.com/open?id=1VATPcbAhnhHZ376_GPixyYAo6u3gN8Os), such as:
+
+- [Logo](https://drive.google.com/open?id=1f5ePg4a00aOrKAXd_AyD05X8MBuJXJiC) 
+- [Fonts](https://drive.google.com/open?id=1sR9XUpUeHY4NfY8Bf1yrfYPN_kvyyOOB)
+- [Video](https://drive.google.com/open?id=187hFrHB0iy8bTZiQ9Pnpt6hb6HKQaMPi)
+
+There're funny resources, also:
+
+- [Ruby Monster's wallpaper](https://drive.google.com/open?id=1lGiPg9pCNHQ-k9j29ZJ_olkZlG0UMLdc)
+
+For HEX code colors and for know what is the better way to use these resources, check the [Nebulab's Brand Guide](https://drive.google.com/open?id=1ia3X-aFuxSVT5hWtYtfdzplF2TfU2Chs).
```

Comment:
*to know how to use these resources
*Nebulab's Brand Guide or Nebulab Brand Guide